packages/fxa-auth-server/lib/senders/emails/templates/subscriptionDowngrade/en.ftl

```diff
@@ -1,10 +1,10 @@
 # Variables:
-# $productNameNew (String) - The name of the subscribed product, e.g. Mozilla VPN
-subscriptionDowngrade-subject = You have switched to { $productNameNew }
+# $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionDowngrade-subject = You have switched to { $productName }
```

Comment:
FWIW we are actually supposed to create new FTL IDs even if only the variable name changes, I checked 😩

However, what I can do instead is put a PR up in the l10n repo to mass replace these just before or after this PR is merged. I confirmed with cocomo that this will work and probably be best.